tests/on_target/README.md

```diff
@@ -16,59 +15,64 @@ docker run --rm -it \
   -v /opt/setup-jlink:/opt/setup-jlink \
   ghcr.io/hello-nrfcloud/firmware:docker-v1.0.3 \
   /bin/bash
+cd thingy91x-oob/tests/on_target
 ```
 
-NOTE: The tests have been tested on Ubuntu 22.04. For details on how to install Docker please refer to the Docker documentation https://docs.docker.com/engine/install/ubuntu/
-
 ### Verify nrfutil/jlink works
 ```shell
 JLinkExe -V
 nrfutil -V
 ```
 
-### Install requirements
-```shell
-cd thingy91x-oob/tests/on_target
-pip install -r requirements.txt --break-system-packages
-```
-
-### Get SEGGER ID
-```shell
-nrfutil device list
-```
-
-### Run UART tests
-
+### NRF91 tests
 Precondition: thingy91x with segger fw on 53
 
+Get device id and set env
```

Comment:
I think we should have to separate shell command here, "get device id" and "set environment variables"